fighters/pacman/src/opff.rs

```diff
@@ -28,9 +28,20 @@ unsafe fn bonus_fruit_toss_ac(boma: &mut BattleObjectModuleAccessor, status_kind
     }
 }
 
+unsafe fn side_special_freefall(fighter: &mut L2CFighterCommon) {
```

Comment:
Why is this in opff
This could just as easily be done in acmd (and probably should be, if you don't want to bother with the status edit)

Reply:
I honestly dont see your point lol, this is a perfect use case for opff (not wanting to edit a convoluted status script)

Reply:
... are you not completely removing its faf by making it go into special fall?

Reply:
Sure, but the point still stands. Don't have to edit motion list this way/avoids changing status in ACMD, which should never be done

Reply:
Fair